deploy/helm/templates/statefulset.yaml

```
@@ -74,6 +74,11 @@ spec:
               containerPort: 662
               protocol: UDP
           securityContext:
+            {{- range $key, $value := .Values.extraArgs }}
+            {{- if and (eq $key "enable-xfs-quota") $value }}
+            privileged: true
+            {{- end }}
+            {{- end }}
             capabilities:
               add:
                 - DAC_READ_SEARCH
```

@mersl sorry for late reply, won't `SYS_ADMIN` privilege be enough for xfs-quota?

```suggestion
            capabilities:
              add:
                {{- range $key, $value := .Values.extraArgs }}
                {{- if and (eq $key "enable-xfs-quota") $value }}
                - SYS_ADMIN
                {{- end }}
                {{- end }}
                - DAC_READ_SEARCH
```